cpp/dolfinx/generation/RectangleMesh.cpp

```diff
@@ -26,6 +27,12 @@ mesh::Mesh build_tri(MPI_Comm comm, const std::array<Eigen::Vector3d, 2>& p,
   {
     Eigen::Array<double, 0, 2, Eigen::RowMajor> geom(0, 2);
     Eigen::Array<std::int64_t, 0, 3, Eigen::RowMajor> topo(0, 3);
+
+    // New mesh builder
```

Comment:
Maybe it could help with later search to add `// TODO` flag here (or some other keyword to identify sketched patterns in the code)?